packages/arb-token-bridge-ui/src/token-bridge-sdk/core/getChainIdFromProvider.ts

```diff
@@ -0,0 +1,6 @@
+import { Provider } from '@ethersproject/providers'
+
+export const getChainIdFromProvider = async (provider: Provider) => {
+  const network = await provider.getNetwork()
```

Comment:
Let's move files around so it can be imported as

```
import { getChainIdFromProvider } from '@/token-bridge-sdk/utils`
```

Reply:
Same for `getAddressFromSigner`

Reply:
Updated